python/taichi/lang/ops.py

```diff
@@ -2,7 +2,7 @@
 import ctypes
 import functools
 import math
-import operator as ops
+import operator as _bt_ops_mod  # bt for builtin
```

Comment:
Previously this has overridden our `ops.py` module, renaming this here for unittests.